ships/botany_bay.json

```diff
@@ -63,7 +63,7 @@
   },
   "ship_ability": {
     "title": "Data Mining",
-    "description": "",
+    "description": "The Botany Bay's Mining Bonus from the Mining Laser is increased by  38050% when mining Corrupted Data and Decoded Data.",
```

Comment:
There's a double space before 38050%

Reply:
I'll parse those out. There seem to be double spaces here and there.